gw-rss-aggregator/src/main/resources/application.yml

```
@@ -0,0 +1,10 @@
+aggregator:
+  bot-token: BOT_TOKEN
```

Еще, вроде, имя бота надо передавать телеграму через `getBotUsername()` (сейчас у вас там пустая строка отдается),

```suggestion
  bot-token: BOT_TOKEN
  bot-name: BOT_NAME
```